src/Workspaces/MSBuildTest/Microsoft.CodeAnalysis.Workspaces.MSBuild.UnitTests.csproj

```diff
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <OutputType>Library</OutputType>
     <RootNamespace>Microsoft.CodeAnalysis.MSBuild.UnitTests</RootNamespace>
-    <TargetFrameworks>$(NetRoslyn);net472</TargetFrameworks>
+    <TargetFrameworks>$(NetVS);net472</TargetFrameworks>
```

Comment:
I wouldn't expect it to be necessary to hold this back. Was there a particular issue when moving this to 9.0?

Reply:
We run the unit tests on a machine with VS installed, so that _might_ matter somehow if maybe the regular SDK isn't there. If that somehow broke, do file a bug @jjonescz and we can investigate further.